priv/repo/seeds.exs

```diff
@@ -1,21 +1,21 @@
 alias CoursePlanner.{Accounts.User, Repo, Settings.SystemVariable}
-import Ecto.Query
 
-has_coordinators? = Repo.one(from u in User, where: u.role == "Coordinator", select: 1, limit: 1)
-unless has_coordinators? do
-  %User{}
-  |> User.changeset(
-      %{
-        name: "first",
-        family_name: "family",
-        email: "testuser@example.com",
-        password: "secret",
-        password_confirmation: "secret",
-        role: "Coordinator"
-      },
-      :seed)
-  |> Repo.insert!
+default_coordinator = Repo.get_by(User, email: "admin@courseplanner.com")
+if default_coordinator do
+  Repo.delete!(default_coordinator)
```

Comment:
why delete it?

Reply:
I don't know, I thought maybe they change the password or something, so I just thought of having some resetting for this user, to ensure it will get created always with the default password.

what do you think about it ?